Migration Portal: v4.5 release (#4072)
EnterpriseDB/docs: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/index.mdx

4.  Enter `yes` or `no` to extract dependent objects and select **OK**. The default is to not extract dependent objects. 

> 5. Enter `yes` or `no` to extract grant statements and select **OK**. The default setting is not to extract grant statements.

Reviewer: ```suggestion
5. Enter `yes` or `no` to extract grant statements and select **OK**. The default setting isn't to extract grant statements.
```
